src/registrar/tests/common.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
```

Comment:
**question:** Is there a meaningful improvement in security if we keep this file outside of `src/` (which is what gets uploaded to CF) and using both a `cp` and the `perl` invocation to turn it on?